package.json

```diff
@@ -30,6 +30,7 @@
   "scripts": {
     "watch": "vitest --watch",
     "test": "vitest",
+    "test:watch": "vitest watch",
```

Comment:
What's the difference between `test:watch` and `watch`?

Reply:
The difference is I can't read and I'm dumb. I thought that was `vite watch`. I'll fix this